core-bundle/src/Command/DebugFragmentsCommand.php

```diff
@@ -86,6 +86,8 @@ static function ($carry, $item): int {
         foreach ($values as $k => $v) {
             if (\is_bool($v)) {
                 $v = $v ? 'true' : 'false';
+            } elseif ('nestedFragments' === $k && \array_key_exists('allowedTypes', $v)) {
+                $v = 'true (allowed types: '.implode(', ', $v['allowedTypes']).')';
```

Comment:
```suggestion
            } elseif (\is_array($v) && array_is_list($v)) {
                $v = implode(', ', $v);
            } elseif (!is_scalar($v)) {
                // For non-list arrays, use YAML notation and fix indentation on multiline strings
                $v = Yaml::dump($v, indent: 2);
                $v = explode("\n", $v);
                array_walk($v, static fn (&$vv, $i) => $vv = $i > 0 ? str_repeat(' ', $length+3).$vv : $vv);
                $v = implode("\n", $v);
```

will render all array values nicely:

<img width="367" alt="Bildschirmfoto 2025-01-16 um 10 21 25" src="https://github.com/user-attachments/assets/c2acec50-1b7e-46c1-8b42-7ae5a5a6b1d2" />